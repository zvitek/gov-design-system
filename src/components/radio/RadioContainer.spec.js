import { shallowMount } from '@vue/test-utils'
import GovRadioContainer from '@components/radio/RadioContainer'

let wrapper

describe('GovRadio', () => {
    beforeEach(() => {
        wrapper = shallowMount(GovRadioContainer)
    })

    it('is called', () => {
        expect(wrapper.name()).toBe('GovRadioContainer')
        expect(wrapper.isVueInstance()).toBeTruthy()
    })

    it('render correctly', () => {
        expect(wrapper.html()).toMatchSnapshot()
    })

    it('should be inline', () => {
        wrapper.setProps({
            inline: true
        })
        expect(wrapper.classes()).toContain('gov-form-group--inline')
    })
})
