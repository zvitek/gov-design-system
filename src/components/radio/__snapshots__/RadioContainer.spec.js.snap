// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GovRadio render correctly 1`] = `<ul class="gov-form-group"></ul>`;
